cedar-policy/src/api.rs

```diff
@@ -290,6 +290,11 @@ impl Default for Authorizer {
 
 impl Authorizer {
     /// Create a new `Authorizer`
+    /// ```
+    /// use cedar_policy::Authorizer;
+    /// let authorizer = Authorizer::new();
+    /// // let r = authorizer.is_authorized(&request, &policy, &entities);
```

Comment:
instead of commenting this line out, can we add hidden lines (to declare `request`, `policy`, `entities`) so that this line works uncommented?

Reply:
done